### ISSUE TITLE
 segmentation_models

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#672]
Segmentation models with pretrained backbones. 

 # Reference
- [ ] [Github repo - keras](https://github.com/qubvel/segmentation_models)
- [ ] [Github repo - segmentation_models.pytorch](https://github.com/qubvel/segmentation_models.pytorch)
- [ ] [Docs](https://segmentation-models.readthedocs.io/en/latest/)
- [ ] jupyter notebook examples - CamVid
  - [ ] [binary_segmentation_camvid.ipynb](https://nbviewer.jupyter.org/github/junxnone/examples/blob/master/segmentation_models/binary_segmentation_camvid.ipynb)
  - [ ] [multiclass_segmentation_camvid.ipynb](https://nbviewer.jupyter.org/github/junxnone/examples/blob/master/segmentation_models/multiclass_segmentation_camvid.ipynb)
- [ ] [Metrics docs](https://segmentation-models.readthedocs.io/en/latest/api.html#metrics)

# Brief
- Based on Keras and TensorFlow Keras.
- Models Arch
  - Unet
  - FPN
  - Linknet
  - PSPNet
- Backbones

Type | Names
-- | --
VGG | 'vgg16' 'vgg19'
ResNet | 'resnet18' 'resnet34' 'resnet50' 'resnet101' 'resnet152'
SE-ResNet | 'seresnet18' 'seresnet34' 'seresnet50' 'seresnet101' 'seresnet152'
ResNeXt | 'resnext50' 'resnext101'
SE-ResNeXt | 'seresnext50' 'seresnext101'
SENet154 | 'senet154'
DenseNet | 'densenet121' 'densenet169' 'densenet201'
Inception | 'inceptionv3' 'inceptionresnetv2'
MobileNet | 'mobilenet' 'mobilenetv2'
EfficientNet | 'efficientnetb0' 'efficientnetb1' 'efficientnetb2' 'efficientnetb3' 'efficientnetb4' 'efficientnetb5' efficientnetb6' efficientnetb7'

# Losses

Loss | Description
-- | --
JaccardLoss | 
DiceLoss |
BinaryCELoss |
CategoricalCELoss |
BinaryFocalLoss |
CategoricalFolcalLoss |

# Metrics

Name | Description
-- | --
IoU Score | ![image](https://user-images.githubusercontent.com/2216970/76181083-e4689980-61fa-11ea-92e7-c3eb2a87842e.png)
F1 Score | ![image](https://user-images.githubusercontent.com/2216970/76181103-f21e1f00-61fa-11ea-9c4a-272fe79b09a7.png) & β=1
F2 Score | ![image](https://user-images.githubusercontent.com/2216970/76181103-f21e1f00-61fa-11ea-9c4a-272fe79b09a7.png) &β=2
Precision |
Recall |

# UseCase
- 二分类
- 多分类
- 多标签分类
